models/book.rb

```diff
@@ -0,0 +1,11 @@
+class Book
+  # Getter and Setters
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
[OPTIONAL] You are getting `:rentals` with the getters method. But you also need a setters method for `:rentals`. Or you have a free chance of making both in one place by using `attr_accessor `